Components/Order/Confirmation.php

```diff
@@ -49,11 +51,18 @@ class Shopware_Plugins_Frontend_NostoTagging_Components_Order_Confirmation
      *
      * @param Shopware\Models\Order\Order $order the order model.
      *
+     * @throws Enlight_Event_Exception
+     * @throws \Doctrine\ORM\ORMException
+     * @throws \Doctrine\ORM\OptimisticLockException
+     * @throws \Doctrine\ORM\TransactionRequiredException
+     * @throws \Nosto\NostoException
      * @see Shopware_Plugins_Frontend_NostoTagging_Bootstrap::onPostUpdateOrder
+     * @suppress PhanDeprecatedFunction
+     * @suppress PhanUndeclaredMethod
      */
     public function sendOrder(Shopware\Models\Order\Order $order)
     {
-        $shop = $order->getShop();
+        $shop = Shopware()->Shop();
```

Comment:
Why not the shop of the order?

Reply:
Because it always returns the shop_id 1 (for German). ```Shopware()-Shop()``` returns the correct ID

Reply:
👍